content/oddnews.njk

```diff
@@ -0,0 +1,54 @@
+---
+title: OddNews
+sub: Because best practice is a conversation...
+banner: OddNews
+summary: |
+  **Hi, OddFriends!** This is the newsletter for designers
+  & developers who want to **get the latest news in your field**
+  — UX Design, CSS, JavaScript or Python — and keep tabs on the full stack.
+---
+
+{% import 'oddnews.macros.njk' as oddnews %}
+{% import 'embed.macros.njk' as embed %}
+{% import 'layout.macros.njk' as layout %}
+
+{{ oddnews.oddnews_signup(
+  heading=false,
+  tag='oddnews',
+  class='oddnews-form-main') }}
+
+{%- set title = 'What to Expect' -%}
+{%- set contents -%}
+  Here are a few examples of topics featured in past editions of OddNews.
+  - **Bird of the Month** - Introduction to the oddest of birds
+  - **CSS** - Miriam Suzanne's latest work in Container Queries
+  - **Python** - How to Run Playwright Inside Docker
+  - **Sass** - How to use the new `split()` function in Sass
+  - **Custom App Development** - Web Apps vs. PWAs vs. Native
+  - **UX Design** - How to conenct designers and developers with Object Oriented UX
+{%- endset -%}
+
+<aside data-layout="cta" class="oddnews-intro has-angles" aria-labelledby="cta-title">
+  {% set callout_content %}
+    <h2 id="cta-title">{{ title | mdInline | safe }}</h2>
+    {{ contents | md | safe }}
+  {% endset %}
+
+  {{ embed.media_block(
+  media=embed.screen(
+  notch='5%',
+  content=embed.img(
+    src='writing/pexels-frank-cone.jpg',
+    alt='An orange hummingbird flying in the air',
+    sizes='media'
+  )
+),
+  content=callout_content
+) }}
+</aside>
+
+{{ layout.title('Latest News') }}
+
+<div class="oddnews-archive">
+  <script language="javascript" src="//oddbird.us19.list-manage.com/generate-js/?u=80219aa68d7bad77b9fd2eb93&fid=27537&show=10" type="text/javascript"></script>
```

Comment:
I take it we don't have control over the contents here? It'd be nice to use a more human-friendly date format, and I don't love their bullet points. But neither of those is a blocker if we can't edit this.

Reply:
Agreed and I double checked again but yea, unfortunately we don't have control over that content 😞 It looks like this is the same block they have on the [public page](https://us19.campaign-archive.com/home/?u=80219aa68d7bad77b9fd2eb93&id=7c27f7fb9a) of the newsletters archive as well.